### DISCUSSION TITLE
Direct Integration Between Web Console and Desktop Application

### DISCUSSION THREAD

**Discussion by Xp96** [rustdesk/rustdesk#9037] [Feature Request]
I would like to suggest a new feature for the RustDesk web console that would enhance user experience and streamline the connection process. I propose a direct integration between the web console and the desktop application using a custom URI scheme.

Here's how it would work:

Generate a unique link: When a user clicks on a device ID in the web console, a unique link with a custom URI scheme (e.g., rustdesk://) would be generated. This link would contain information about the device, such as its ID and host.
Intercept the link: The RustDesk desktop application would be configured to intercept this custom URI scheme. When the user clicks the generated link, the desktop app would automatically open and establish a connection to the specified device.
![image](https://github.com/user-attachments/assets/b2c6e17b-0f3f-4343-ab12-f858c1e9eb20)

**Comment by djordans** [comment]
Hi,

Rustdesk://9123456234@192.168.16.1:21117?key=5Qbwsde3unUcJBtrx9ZkvUmwFNoExHzpryHuPUdqlWM=
